src/definitions/elements/icon.less

```diff
@@ -372,4 +372,49 @@ i.icons {
   }
 }
 
+& when ((@variationIconGroups) or (@variationIconCorner)) and (@variationIconBordered) {
+  i.bordered.icons {
+    width: @borderedSize;
+    height: @borderedSize;
+    box-shadow: @borderedShadow;
+  }
+  i.bordered.icons i.icon:first-child {
+    position: absolute;
+    transform: translateX(-50%) translateY(-50%);
+  }
+
+  & when (@variationIconCorner) {
+    i.bordered.icons .corner.icon {
+      top: auto;
+      left: auto;
+      right: .8em;
+      bottom: .8em;
+    }
+    i.bordered.icons .icon.corner[class*="top right"] {
+      top: .8em;
+      left: auto;
+      right: .8em;
+      bottom: auto;
+    }
+    i.bordered.icons .icon.corner[class*="top left"] {
+      top: .8em;
+      left: .8em;
+      right: auto;
+      bottom: auto;
+    }
+    i.bordered.icons .icon.corner[class*="bottom left"] {
+      top: auto;
+      left: .8em;
+      right: auto;
+      bottom: .8em;
+    }
+    i.bordered.icons .icon.corner[class*="bottom right"] {
+      top: auto;
+      left: auto;
+      right: .8em;
+      bottom: .8em;
+    }
```

Comment:
Are these position unit (.8em) fit for the other icon size? In your test case, you only use for huge icons. So, I'm not sure if you forgot to test with the other icon sizes.

Can you please also provide JS fiddle? So, we can test your changes easily.

Reply:
I'm not sure how do to the jsfiddle correctly in this case, so I copied in the output from the complied less code into it.
https://jsfiddle.net/fpb9qLu1/

`.8em` works fine in all sizes.